docs/guides/primitives.mdx

```diff
@@ -81,25 +80,46 @@ sampler.run([
 ])
 ```
 
-## How to use Qiskit primitives
+## Benefits of Qiskit primitives
+
+For Qiskit users, primitives let you write quantum code for a specific QPU without having to explicitly
+manage every detail. Also, because of the additional layer of abstraction, you might be able to more easily
+access advanced hardware capabilities of a given provider. For example, with Qiskit Runtime primitives,
+you can leverage the latest advancements in error mitigation and suppression by toggling options such as the primitive's [`resilience_level`,](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions#resilience_level) rather than building your own implementation of these techniques.
```

Comment:
```suggestion
With primitives, Qiskit users can write quantum code for a specific QPU without having to explicitly
manage every detail. Also, because of the additional layer of abstraction, you might be able to more easily
access advanced hardware capabilities of a given provider. For example, with Qiskit Runtime primitives,
you can take advantage of the latest advancements in error mitigation and suppression by toggling options such as the primitive's [`resilience_level`,](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.EstimatorOptions#resilience_level) rather than building your own implementation of these techniques.
```